packages/multiselect/src/react/index.tsx

```diff
@@ -1,18 +1,23 @@
 import { CaretDownIcon, CloseIcon } from '@pluralsight/ps-design-system-icon'
-import Field from '@pluralsight/ps-design-system-field'
+import { Field } from '@pluralsight/ps-design-system-field'
 import { BelowLeft } from '@pluralsight/ps-design-system-position'
-import Tag from '@pluralsight/ps-design-system-tag'
+import { Tag } from '@pluralsight/ps-design-system-tag'
 import { HTMLPropsFor, canUseDOM } from '@pluralsight/ps-design-system-util'
 import { useCombobox, useMultipleSelection } from 'downshift'
 import glamorDefault, * as glamorExports from 'glamor'
 import React from 'react'
 
 import stylesheet from '../css/index'
 import { Menu } from './menu'
-import { FilterFn, OnStateChangeFn, Option, StateReducer } from './types'
+import {
+  FilterFn,
+  OnStateChangeFn,
+  MultiSelectOption,
+  StateReducer
+} from './types'
 import { noop, simpleTextFilter, switchcase } from './utils'
 
-export { Option }
+export { MultiSelectOption }
```

Comment:
Why is this wrapped in curlies?

Reply:
I was refactoring the name not how it had already been coded

Reply:
Fair enough. I was asking because I wasn't sure about the syntax generally and hoped you'd be able to explain to me how it works. Is this a short-hand for exporting local vars?

Reply:
oh that yea it is. If you import a variable use and also want to re-export it.